### PR TITLE
Add support for restricting helix log queries to a specified work item name

### PR DESCRIPTION
**Pull request by dibarbet**
Includes the HelixWorkItemName in the test results table so that we can query only for matching work items when searching helix logs.  This allows us to track crashes in work items containing `Microsoft.CodeAnalysis.EditorFeatures.UnitTests.dll` with separate issues from crashes in other work items which can be a different problem.

I tested this on the test database and perform the ef db update there.

What this looks like:

Searching for logs in work items with "class"
image

Searching for logs in work items with "jit"
image

### CODE REVIEW

File: DevOps.Util.DotNet/Triage/Model.Query.cs
Comment (jaredpar) [inline DevOps.Util.DotNet/Triage/Model.Query.cs:82]: Is there a reason you didn't update the search tests results to also allow querying for this property?

Comment (dibarbet) [reply DevOps.Util.DotNet/Triage/Model.Query.cs:82]: Hmm, I didn't really need it, but I can see it being useful (e.g. want to only query for test failures for a specific work item name).  

I think it uses the same indices so it should be easy for me to add it, so I'll take a look.

Comment (dibarbet) [reply DevOps.Util.DotNet/Triage/Model.Query.cs:82]: updated
image

**Review by jaredpar — APPROVED** [review]

**Comment by jaredpar** [comment]
Thanks for getting this done! 

I'm going to wait until this evening to merge the changes. I have to update the indexes at the same time I merge this and that is likely to take ~5-20 min to complete. Want to do it this evening when the build throughput is lower so there is less disruption.

**Comment by dibarbet** [comment]
Works for me, feel free to ping me if you hit any issues with it, I should be available this evening

**Pull request merged**
